macheteLogger/src/main/java/com/virtuslab/logger/MacheteLogger.java

```diff
@@ -5,30 +5,73 @@
 import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
 
 public class MacheteLogger implements IMacheteLogger {
-  private final LambdaLogger LOG;
+  private final LambdaLogger logger;
   private final String className;
 
   MacheteLogger(LambdaLogger logger) {
-    LOG = logger;
-    // We are sure that at the moment when this constructor is called we have at least 4 elements in stacktrace array
+    this.logger = logger;
+    // We are sure that at the moment when this constructor is called we have at least 4 elements in stacktrace array:
+    // #0: java.lang.Thread#getStackTrace
+    // #1: com.virtuslab.logger.MacheteLogger#<init>
+    // #2: com.virtuslab.logger.MacheteLoggerFactory#getLogger
+    // #3: method from MacheteLoggerFactory#getLogger was called
```

Comment:
`s/from/from which`; btw isn't it typically the static initializer of the class aka `<clinit>` rather than a "proper" method?

Reply:
TBH I don't know - this stacktrace is what I got from program run  xD

Reply:
Yeah, but the result of `MacheteLoggerFactory#getLogger` is typically assigned to a static field (as opposed to e.g. being called from instance/static method)... and static fields are initialized in a synthetic method call `<clinit>`